src/expint.jl

```diff
@@ -297,21 +297,46 @@ end
 # series about origin, general ν
 # https://functions.wolfram.com/GammaBetaErf/ExpIntegralE/06/01/04/01/01/0003/
 function En_expand_origin_general(ν::Number, z::Number)
-    gammaterm = En_safe_gamma_term(ν, z)
+    # gammaterm = En_safe_gamma_term(ν, z)
+    gammaterm = gamma(1-ν)*z^(ν-1)
     frac = one(real(z))
     sumterm = frac / (1 - ν)
     k, maxiter = 1, 100
     ϵ = 10*eps(real(sumterm))
+    blowup = 0
+
     while k < maxiter
         frac *= -z / k
         prev = sumterm
-        sumterm += frac / (k + 1 - ν)
-        if abs(sumterm - prev) < ϵ
-            break
+        if abs(k + 1 - ν) < 0.5
+            blowup += frac / (k + 1 - ν)
+        else
+            sumterm += frac / (k + 1 - ν)
+            if abs(sumterm - prev) < ϵ
+                break
+            end
         end
         k += 1
     end
-    return gammaterm - sumterm
+    
+    if isreal(round(ν)) && real(round(ν)) > 0
```

Comment:
Seems like this should only be done in double (or single) precision since we don't have a general series?